joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/utils/NodeBuilders.scala

```diff
@@ -0,0 +1,30 @@
+package io.joern.x2cpg.utils
+
+import io.shiftleft.codepropertygraph.generated.nodes.{NewCall, NewIdentifier}
+import io.shiftleft.codepropertygraph.generated.{DispatchTypes, Operators}
+
+object NodeBuilders {
+  def assignmentNode(): NewCall = NewCall()
+    .name(Operators.assignment)
+    .methodFullName(Operators.assignment)
+    .dispatchType(DispatchTypes.STATIC_DISPATCH)
+
+  def indexAccessNode(): NewCall = NewCall()
+    .name(Operators.indexAccess)
+    .methodFullName(Operators.indexAccess)
+    .dispatchType(DispatchTypes.STATIC_DISPATCH)
+
+  def identifierNode(
+    name: String,
+    typeFullName: String,
+    line: Option[Integer] = None,
+    column: Option[Integer] = None,
+    dynamicTypeHintFullName: Seq[String] = Seq.empty
+  ): NewIdentifier = NewIdentifier()
+    .name(name)
+    .code(name)
+    .typeFullName(typeFullName)
+    .lineNumber(line)
+    .columnNumber(column)
+    .dynamicTypeHintFullName(dynamicTypeHintFullName)
+}
```

Comment:
I would have expected those methods to be also in `AstCreatorBase`. Is there a reasons why you did not do that?